docs/monitoring/inventory.md

```diff
@@ -1,13 +1,107 @@
 title: Sematext Inventory Monitoring
 description: Sematext Inventory Monitoring gives you insight into your whole infrastructure automatically based on the applications, containers, servers and general infrastructure you are monitoring.
 
-This should explain the new Inventory Monitoring feature, and have the following sections.
+##Overview
 
-- overview
-- how it works
-- how to enable
-- what it does
-- what it solves 
-- how it helps with observability/troubleshooting/debugging/etc…
+Sematext Monitoring Agent provides simple to use and versatile way of gathering machine related information such as host, VM or container physical properties, kernel versions, installed packages and presents them on per host basis allowing you not only to view the data but also search and compare different hosts together. Everything in a single place shipped automatically without any effort from the operations point of view.
```

Comment:
Sematext Monitoring Agent provides simple to use and versatile way of gathering machine related information such as host, VM or container physical properties, kernel versions, installed packages and presents them on per host basis allowing you not only to view the data but also search and compare different hosts together.
->
The [Sematext Agent] (add a link to the STA docs page) provides a simple and versatile way of gathering machine-related information such as host, VM, or container properties, kernel versions, and installed packages. It presents them on a per-host basis allowing you not only to view the data but also search and compare different hosts.

Everything in a single place shipped automatically without any effort from the operations point of view.
->
All of this, in the same place, shipped automatically without any effort from an operations point of view.

Reply:
Done.